CMakeLists.txt

```
@@ -1,5 +1,5 @@
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.20)
```

```suggestion
cmake_minimum_required(VERSION 3.5)
```
It's sufficient to update only to 3.5 (released in 2016), versus 3.20 (released in 2021). Ubuntu 20.04LTS uses CMake 3.16, for example.